modules/orbit/andes/pom.xml

```diff
@@ -145,6 +145,7 @@
             com.google.common.base;version="19.0.0",
             com.google.common.util.concurrent;version="19.0.0",
             com.google.common.cache;version="19.0.0",
+            org.wso2.carbon.business.messaging.core.authentication,
```

Comment:
A bit confused here. cbm is a dependency of andes as I know. So when releasing andes we to build cbm ? Or am I wrong ? :)

Reply:
Yes, It's a circular dependency. Andes need CBM and CBM needs Andes. Something that needs to be fixed before release.